LanguageFeatures/Patterns/patterns_lib.dart

```diff
@@ -84,70 +96,97 @@ class Shape {
     }
   }
 
-  Unit get area {
+  Unit<T> get area {
     _log("Shape.area");
     return Unit(0, logFunction);
   }
 
-  Unit get size {
+  Unit<T> get size {
     _log("Shape.size");
     return Unit(0, logFunction);
   }
+
+  double get areaAsDouble => 0;
+  double get sizeAsDouble => 0;
+  int get areaAsInt => areaAsDouble.toInt();
+  int get sizeAsInt => sizeAsDouble.toInt();
+  num? get areaAsNullable => areaAsDouble;
+  num? get sizeAsNullable => sizeAsDouble;
+  num? get areaAsNull => null;
+  num? get sizeAsNull => null;
+  List<num> get areaAsList => [areaAsDouble];
+  Map<String, num> get areaAsMap => {Area: areaAsDouble};
```

Comment:
Ah, it's a constant variable. But then it is definitely recommended to spell it as `area`.

Reply:
We have a getter named `area` in this class. If this constant is confusing let's get rid of it and use `"area"`. This will made the code more readable

Reply:
Sounds good!